tests/__mocks__/@octokit/graphql-schema.ts

```diff
@@ -0,0 +1,3 @@
+module.exports = {
+    validate: () => [],
```

Comment:
Is this just to mock the module?
Doesn't seem properly indented.

Reply:
Yeah Jest fails to import it. My format-on-save was forcing the indent issue for some reason